src/xenia/kernel/xam/xam_ui.cc

```diff
@@ -575,6 +575,23 @@ dword_result_t XamShowDeviceSelectorUI_entry(
     dword_t user_index, dword_t content_type, dword_t content_flags,
     qword_t total_requested, lpdword_t device_id_ptr,
     pointer_t<XAM_OVERLAPPED> overlapped) {
+  if (!overlapped) {
+    return X_ERROR_INVALID_PARAMETER;
+  }
+
+  if ((user_index >= 4 && user_index != 0xFF) ||
+      (content_flags & 0x83F00008) != 0 || !device_id_ptr) {
+    kernel_state()->CompleteOverlappedImmediate(overlapped,
```

Comment:
Use: ``XOverlappedSetExtendedError(overlapped, X_ERROR_INVALID_PARAMETER);``

I see that this is only field that is modified in such case ``CompleteOverlappedImmediate`` modified all of them accordingly, which might not be expected